.babelrc

```diff
@@ -1,8 +1,3 @@
 {
-  "plugins": [
-    "transform-class-properties",
-    "transform-object-assign",
-    "transform-object-rest-spread"
-  ],
-  "presets": ["es2015", "react"]
+  "presets": ["env", "react"]
```

Comment:
Is this env preset helpful even without configuration (like target browser and environments)?